build.gradle

```diff
@@ -138,34 +159,12 @@ task copyDeps(type: Copy) {
 }
 
 runIde {
-    jvmArgs '--add-exports', 'java.base/jdk.internal.vm=ALL-UNNAMED'
-}
-
-intellij {
-    // For a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
-    version = '2024.1.6'
-
-    def lsp4ij = project.hasProperty('useLocal') && project.property('useLocal') == 'true' ?
-            file("../lsp4ij/build/distributions/LSP4IJ/") :
-            'com.redhat.devtools.lsp4ij:' + lsp4ijVersion
-    plugins = ['java', 'maven', 'gradle-java', 'properties', 'terminal', 'org.jetbrains.idea.maven', 'com.intellij.gradle', lsp4ij]
-    updateSinceUntilBuild = false
-}
-
-downloadRobotServerPlugin {
-    version.set(remoteRobotVersion)
-}
-
-runIdeForUiTests {
```

Comment:
Similar to the unit test framework, we should not be removing the `runIdeForUiTests` or `downloadRobotServerPlugin` sections. This should be left alone for now.

Reply:
I removed this part, as @vaisakhkannan will be raising the PR for UI test part including all this. If you think I shouldn't remove and comment it out I can do so.

Reply:
Please do not remove. Let @vaisakhkannan handle it in his PR.

Reply:
Sure, I will add back the changes.